BigBaseV2/src/main.cpp

```diff
@@ -76,6 +77,7 @@ BOOL APIENTRY DllMain(HMODULE hmod, DWORD reason, PVOID)
 					auto notification_service_instance = std::make_unique<notification_service>();
 					auto pickup_service_instance = std::make_unique<pickup_service>();
 					auto player_service_instance = std::make_unique<player_service>();
+					auto gta_data_service_instance = std::make_unique<gta_data_service>();
```

Comment:
Call reset for this instance when unloading the DLL, see the other services that are beind called with .reset() further down.